src/pathwayvis/components/legend/legend.component.ts

```diff
@@ -1,9 +1,34 @@
 import * as template from './legend.component.html';
 import * as angular from "angular";
 import './legend.component.scss';
+import * as _ from 'lodash';
+import { MapOptionService } from "../../services/mapoption.service";
+
+
+const defaultColor = 'white';
+const warningColor = '#FEEFB3';
+
+
+class LegendComponentCtrl {
+    private _mapOptions: MapOptionService;
+    private background: string;
+
+    constructor($scope: angular.IScope,
+                mapOptions: MapOptionService) {
+        this._mapOptions = mapOptions;
+        this.background = defaultColor;
+    }
+
+    private getPredictedGrowth(): number {
+      let rate = _.round(this._mapOptions.getCurrentGrowthRate(), 3);
```

Comment:
js does rounding, over the long term to plan is to eliminate lodash, Could we switch to this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Number/toPrecision?